Makefile

```diff
@@ -197,7 +197,7 @@ coverage-report:
 .PHONY: integration-tests
 integration-tests: $(TERRAFORM_BIN) $(KUBECTL_BIN) ## Start test environment, run acceptance tests, generate coverage report, and teardown test environment
 	$(MAKE) deploy-test-helper
-	KUBECONFIG=$(OUTPUT_DIR)/kubeconfig.yml $(MAKE) testacc
```

Comment:
Does it make sense to only set `NUODB_CP_URL_BASE` if it is not already set? We are already letting through `NUODB_CP_USER` and `NUODB_CP_PASSWORD`, if the user sets them (which they should not), so it is weird to only override one of the settings.